src/routes/safe/components/AddressBook/index.tsx

```diff
@@ -50,7 +50,7 @@ const AddressBookTable = ({ classes }) => {
   const safesList = useSelector(safesListSelector)
   const entryAddressToEditOrCreateNew = useSelector(addressBookQueryParamsSelector)
   const addressBook = useSelector(getAddressBook)
-  const [selectedEntry, setSelectedEntry] = useState(null)
+  const [selectedEntry, setSelectedEntry] = useState<any>(null)
```

Comment:
In this case ```any``` is better than untyped?

Reply:
it won't compile if untyped and I didn't want to dig into typing because the component is complex